tests/recordings/-pollyjs_2160168770/adapter-fetch_162589637/Integration-Fetch-Adapter_1763210894/should-be-able-to-abort-from-an-intercept_68559697/recording.har

```diff
@@ -6,6 +6,7 @@
       "version": "67.0"
     },
     "creator": {
+      "comment": "persister:rest",
```

Comment:
Not sure if we want this or something like `via the rest persister`...

Reply:
this is fine.  just needs some test coverage